honeycomb/honeycomb.go

```diff
@@ -499,6 +511,10 @@ func (e *Exporter) ExportSpan(ctx context.Context, data *trace.SpanData) {
 		ev.AddField("error", true)
 	}
 
+	// Treat resource-defined attributes as overlays, taking precedent over any same-keyed span
+	// attributes. Apply them last.
+	transcribeResourceTo(ev)
```

Comment:
Should we move this up before we set the "status.code" and "status.message" attributes, so that those can't be overridden by same-keyed attributes in a resource?

Reply:
I went ahead and did that.